app/assets/stylesheets/nollning.scss

```diff
@@ -0,0 +1,33 @@
+$top: "theme/top.svg";
+$pillars: "theme/pillars.svg";
+$stairs: "theme/stairs.svg";
+.roman-top {
+  border: 90px solid transparent;
+  padding-top: 25px;
+  -webkit-border-image: url($top) 40% 40% 40% 40% repeat;
+  -o-border-image: url($top) 40% 40% 40% 40% repeat;
+  border-image: url($top) 40% 40% 40% 40% repeat;
+}
+.roman-pillars {
+  border-top: 20px solid transparent;
+  border-bottom: 20px solid transparent;
+  border-left: 110px solid transparent;
+  border-right: 110px solid transparent;
+  -webkit-border-image: url($pillars) 40% 40% 40% 40% repeat;
+  -o-border-image: url($pillars) 40% 40% 40% 40% repeat;
+  border-image: url($pillars) 30% 37% 30% 37% repeat;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+.roman-footer {
+  border-top: 110px solid transparent;
```

Comment:
Properties should be ordered border-bottom, -o-border-image, -webkit-border-image, border-image, border-left, border-right, border-top, margin-left, margin-right, margin-top